lib/osf-components/addon/components/file-browser/file-rename-modal/component.ts

```diff
@@ -51,6 +66,7 @@ export default class FileRenameModal extends Component<Args> {
         } catch(e) {
             this.toast.error(this.intl.t('osf-components.file-browser.file_rename_modal.retry_message'));
         }
+        this.updateOriginalName(newName);
```

Comment:
With the pending changes for https://github.com/CenterForOpenScience/ember-osf-web/pull/1546 this could be removed.